src/scripts/activate-user.ts

```diff
@@ -12,6 +14,19 @@ import * as Console from 'fp-ts/lib/Console';
 export const activateUser = (
   service: UserService,
   userId: string
+): TE.TaskEither<string, User> =>
+  pipe(
+    getUser(service, userId),
+    TE.chain((user) => checkUserStatusPriorToActivation(user)),
+    TE.chain((user) => service.activateUser(user.id)),
+    TE.chainFirstIOK((user) =>
```

Comment:
**style:** prefer https://gcanti.github.io/fp-ts/modules/TaskEither.ts.html#tapio to `chainFirstIOK` for new code in this code base.

Reply:
updated in commit: expire-password-and-get-temporary-password